src/features/page_view_timing/aggregate/index.js

```diff
@@ -44,12 +45,24 @@ export class Aggregate extends AggregateBase {
     })
 
     /* First Contentful Paint - As of WV v3, it still imperfectly tries to detect document vis state asap and isn't supposed to report if page starts hidden. */
-    onFCP(({ name, value }) => {
-      if (pageStartedHidden || this.alreadySent.has(name)) return
-      this.alreadySent.add(name)
-
-      this.addTiming(name.toLowerCase(), value)
-    })
+    if (iOS_below16) {
+      try {
+        if (!pageStartedHidden) { // see ios-version.js for detail on this following bug case; tldr: buffered flag doesn't work but getEntriesByType does
+          const paintEntries = performance.getEntriesByType('paint')
+          paintEntries.forEach(entry => {
+            if (entry.name === 'first-contentful-paint') {
+              this.addTiming('fcp', Math.floor(entry.startTime))
```

Comment:
Do we need to capture `fp` here as well?

Reply:
`first-paint` wasn't (and may still not be) supported in safari (or at least our past testing), so going to be a nope.